app/src/test/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewInitTest.kt

```diff
@@ -0,0 +1,25 @@
+package org.simple.clinic.summary.bloodpressures.newbpsummary
+
+import com.spotify.mobius.test.FirstMatchers.hasEffects
+import com.spotify.mobius.test.FirstMatchers.hasModel
+import com.spotify.mobius.test.InitSpec
+import com.spotify.mobius.test.InitSpec.assertThatFirst
+import org.junit.Test
+import java.util.UUID
+
+class NewBloodPressureSummaryViewInitTest {
+  @Test
+  fun `when widget is created, then load blood pressures for patient`() {
+    val patientUuid = UUID.fromString("33d7f6d7-39f9-46a2-a104-457e5f77dc20")
+    val defaultModel = NewBloodPressureSummaryViewModel.create(patientUuid)
+    val initSpec = InitSpec<NewBloodPressureSummaryViewModel, NewBloodPressureSummaryViewEffect>(NewBloodPressureSummaryViewInit())
```

Comment:
Can you add a line break here so that there's a clear separation between the variable declarations and the actual test setup please?